docs/developers_guide.md

```diff
@@ -183,7 +183,7 @@ Checkout [this](https://www.scalyr.com/blog/zipkin-tutorial-distributed-tracing/
 of our [fork](https://github.com/ease-lab/firecracker-containerd) of the upstream repository.
 Currently, we are in the process of upstreaming VM snapshots support to the upstream repository.
 
-* Current Firecracker version is 0.24.0, Knative 0.26.0, Kubernetes 1.20.6, gVisor 20210622.0, and Istio 1.9.5.
+* Current Firecracker version is 0.24.0, Knative 1.3, Kubernetes 1.23.5, gVisor 20210622.0, and Istio 1.12.5.
```

Comment:
It seems you also updated gVisor in `install_stock.sh` to 20220117. Please check if it's the latest version and if the tests pass